packages/block-editor/src/components/block-list/style.scss

```diff
@@ -211,6 +211,10 @@
 		}
 	}
 
+	&.is-outline-mode.is-selected.rich-text {
+		cursor: unset;
```

Comment:
Do you know what style this is overriding? Maybe the fix is to actually remove a style (or be more specific there)

Reply:
I believe [this](https://github.com/WordPress/gutenberg/blob/5b60168b75629f08d8ec29bb582291ac13d55413/packages/block-editor/src/components/block-list/style.scss#L219) is the style being overridden.

Since we want to retain the `default` cursor on _hover_, and expose the `text` cursor on _selection_, I'm not sure how we can be more efficient here.

Reply:
So we want to retain the default on "hover" unless the block is focused?

Reply:
To expand a bit, I don't think we should be targeting "rich-text" here, that's just an implementation detail of the block, there's no guarantee the block is using that class. We should instead use the blocks states

Reply:
Got it, I'll try another approach.